crates/evm/src/fuzz/invariant/executor.rs

```diff
@@ -19,7 +19,8 @@ use crate::{
     utils::get_function,
     CALLER,
 };
-use ethers::abi::{Abi, Address, Detokenize, FixedBytes, Tokenizable, TokenizableItem};
```

Comment:
We use these traits a bunch—what would be the best way to replace them? This also ties in with the main executor, which uses `Detokenize` + `Tokenize` a bunch.